redaxo/src/addons/structure/lib/structure_element.php

```diff
@@ -96,6 +96,24 @@ public function getValue($value)
         return null;
     }
 
+    /**
+     * Returns the value from this element or from the closest parent where the value is set.
+     *
+     * @return string|int|null
+     */
+    public function getClosestValue(string $value)
+    {
+        $value = $this->getValue($value);
+
+        if (null !== $value && '' !== $value) {
+            return $value;
+        }
+
+        $parent = $this->getParent();
+
+        return $parent ? $parent->getClosestValue($value) : null;
```

Comment:
Falls wir getClosest behalten, könnte ich die Methode jetzt auch damit implementieren:

```suggestion
        $closest = $this->getClosest(function (self $element) use ($value) {
            return !in_array($element->getValue($value), [null, ''], true);
        });

        return $closest ? $closest->getValue($value) : null;
```

Wobei ich mir unsicher bin, ob das schöner ist?

Reply:
Würde ich nicht machen, sondern so lassen